dockstore-client/src/main/java/io/dockstore/client/cli/ArgumentUtility.java

```diff
@@ -57,11 +58,19 @@ public static void out(String arg) {
     }
 
     public static void err(String arg) {
-        LOG.error(arg);
+        if (LOG.isInfoEnabled()) {
+            LOG.error(arg);
+        } else {
+            System.err.println(ERROR_MESSAGE_START + arg);
```

Comment:
Use io.dockstore.client.cli.ArgumentUtility#err for consistency.

Reply:
This is where `err` is defined. Would it better to call this method, `printError` and define `err` to be `System.err.println`?

Reply:
🤦 Ignore my comment.